state-transition/core/state_processor_staking_test.go

```diff
@@ -726,31 +726,9 @@ func TestTransitionHittingValidatorsCap_ExtraSmall(t *testing.T) {
 	blk = moveToEndOfEpoch(t, blk, cs, sp, st, ctx, depRoot)
 
 	// finally the block turning epoch. extra validator deposits
-	// will be withdrawn within 3 blocks (#Validator / MaxValidatorsPerWithdrawalsSweep)
```

Comment:
the exact point at which withdrawal is made may way (within the right epoch). Same for test below